src/assessor_tools.yml

```
@@ -294,6 +316,19 @@
         powershell: yes
         unarchive_extra_opts:
           - --strip-components=1
+    - role: assessment_tool
+      vars:
+        archive_src: https://github.com/skelsec/pypykatz/tarball/master
+        install_dir: /tools/pypykatz
+        pip_packages:
+          - pypykatz
+          - minidump
+          - minikerberos
+          - aiowinreg
+          - msldap
+          - winacl
```

Please sort this list alphabetically:
```suggestion
          - aiowinreg
          - minidump
          - minikerberos
          - msldap
          - pypykatz
          - winacl
```